Carmu/Sources/Views/SessionStart/SessionStartDriverView.swift

```diff
@@ -123,6 +123,17 @@ final class DriverFrontView: UIView {
         return view
     }()
 
+    lazy var crewCollectionView: UICollectionView = {
+        let layout = UICollectionViewFlowLayout()
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collectionView.backgroundColor = UIColor.semantic.backgroundSecond
+        collectionView.layer.cornerRadius = 16
+        return collectionView
+    }()
+    private let crewCollectionViewCellIdentifier = "crewCell"
```

Comment:
CrewCollectionViewCell에 대한 식별자라면 CrewCollectionViewCell 클래스 내부에서 static 변수로 갖고 있는게 더 관리하기 편하지 않을까 싶군요🤔
```suggestion
// CrewCollectionViewCell.swift
final class CrewCollectionViewCell: UICollectionViewCell {
    ...
    static let cellIdentifier = "crewCell"
}
```

Reply:
오호 참고하도록 하겠슴당!